Yafc.Model/Yafc.Model.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <RootNamespace>Yafc.Model</RootNamespace>
```

Comment:
Was this change automatic? If not, why was this line moved?

Reply:
I'm not sure why this line shows as removed, it's not removed.
I did move the TargetFramework around to make it match the same order in al files.

These lines can be removed, but i haven't done so because i want to keep that for if we ever do a all file namespace refactoring. That's outside the scope of this pr. (Namespaces should match folder structure.)

Reply:
The thing that I wanted to clarify is if `<TargetFramework>` being moved above `<RootNamespace>` was done manually or automatically. Before the change the TargetFramework was below, so I am curious if it was moved automatically.

Reply:
Yes, as i said:

> I did move the TargetFramework around to make it match the same order in al files.

If reordering is inappropriate for this pr i can revert it, but i found keeping the order consistent between files easier to work with.

Reply:
If the order is the same in the other files, then it's fine.